Added target metadata examples for trust-manager

terricain opened the pull request:
A follow up from https://github.com/cert-manager/trust-manager/pull/582 which adds the options to specify annotations and labels in the target configmaps/secrets that a Bundle creates.

Do I need to update `content/docs/trust/trust-manager/api-reference.md` using the latest commit from trust-manager's main branch?

terricain commented:
/cc erikgb

erikgb commented:
Thanks for following up on this, @terricain!

> Do I need to update content/docs/trust/trust-manager/api-reference.md using the latest commit from trust-manager's main branch?

Well, ideally yes. 👍 These API docs seem to be updated manually, which is a pity. I don't know how we could have automated this. Any idea @inteon @SgtCoDFish?

terricain commented:
Have tweaked the `generate-trust-manager` script to optionally take a git reference (also ran `shellcheck` over it and added some quotes in there). API reference is now updated 😄

erikgb reviewed (commented):
Great!

/approve
/cc @SgtCoDFish 

Ashley, would it be ok to merge this now? Or should we wait for a new release of trust-manager?

inteon reviewed (approved):
Thanks for also regenerating the API ref.
/approve
/lgtm
